noetic/distribution.yaml

```diff
@@ -7226,6 +7226,21 @@ repositories:
       url: https://github.com/wxmerkt/pybind11_catkin-release.git
       version: 2.5.0-1
     status: maintained
+  pyhri:
+    doc:
+      type: git
+      url: https://github.com/ros4hri/pyhri.git
+      version: master
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/ros4hri/pyhri-release.git
+      version: 0.1.2-1
+    source:
+      type: git
+      url: https://github.com/ros4hri/pyhri.git
```

Comment:
Please add a LICENSE file to the source repository (see our [review guidelines](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md#types-of-pull-requests)).
A new release is not necessary, just the change to the source repository.

Reply:
done. Thanks!

Reply:
Sorry, I don't see a LICENSE file here yet: https://github.com/ros4hri/pyhri

Reply:
oops, wrong push. Fixed now.